conftest.py

```diff
@@ -9,11 +9,13 @@ def pytest_addoption(parser):
     parser.addoption("--ignore_machine_config",
                      action='store_true',
                      help="Disable checks/assertions for machine configuration for stable benchmarks")
-    parser.addoption("--disable_nograd", action='store_true', 
+    parser.addoption("--disable_nograd", action='store_true',
                     help="Disable no_grad for eval() runs")
     parser.addoption("--check_opt_vs_noopt_jit",
                      action='store_true',
                      help="The best attempt to check results for inference runs. Not all models support this!")
+    parser.addoption("--localrun", action='store_true',
```

Comment:
Since the "--localrun" only runs cpu benchmarks, maybe it makes more sense to rename it to "--cpuonly". Also, please mention in the help and doc that specifying this option will automatically ignore the machine config check.